lib/rubyblok/mixins/webhook.rb

```diff
@@ -4,10 +4,23 @@ module Webhook
       extend ActiveSupport::Concern
 
       included do
+        skip_before_action :verify_authenticity_token
```

Comment:
didn't we use the `only: create` option here?

Reply:
I thought that was not necessary, because I added in the routes:
```
def routes_content
    "\n  resources :#{file_name}, only: :create" 
end
```